src/main/java/org/aion/wallet/connector/core/CoreBlockchainConnector.java

```diff
@@ -43,15 +50,19 @@ public void createAccount(final String password, final String name) {
     }
 
     @Override
-    protected String sendTransactionInternal(SendRequestDTO dto) throws ValidationException {
-        if (!unlock(dto)) {
-            throw new ValidationException("Failed to unlock wallet");
-        }
-        ArgTxCall transactionParams = new ArgTxCall(Address.wrap(ByteUtil.hexStringToBytes(dto.getFrom()))
-                , Address.wrap(ByteUtil.hexStringToBytes(dto.getTo())), dto.getData(),
-                dto.getNonce(), dto.getValue(), dto.getNrg(), dto.getNrgPrice());
+    public AccountDTO addKeystoreUTCFile(byte[] file, String password, final boolean shouldKeep) throws ValidationException {
+        throw new ValidationException("Unsupported operation");
+    }
 
-        return TypeConverter.toJsonHex(API.sendTransaction(transactionParams));
+    @Override
+    public AccountDTO addPrivateKey(byte[] raw, String password, final boolean shouldKeep) {
+        return null;
```

Comment:
this is not necessarily wanted
maybe throw an Unimplemented exception or something similar